portable-atomic-util/src/arc.rs

```diff
@@ -77,11 +79,13 @@ macro_rules! acquire {
 /// This is an equivalent to [`std::sync::Arc`], but using [portable-atomic] for synchronization.
 /// See the documentation for [`std::sync::Arc`] for more details.
 ///
-/// **Note:** Unlike `std::sync::Arc`, coercing `Arc<T>` to `Arc<U>` is not supported at all.
+/// **Note:** Unlike `std::sync::Arc`, coercing `Arc<T>` to `Arc<U>` is only possible if
+/// the optional cfg `portable_atomic_unstable_coerce_unsized` is enabled, as documented for this crate,
+/// and this optional cfg item is only supported with Rust nightly version.
 /// This is because coercing the pointee requires the
 /// [unstable `CoerceUnsized` trait](https://doc.rust-lang.org/nightly/core/ops/trait.CoerceUnsized.html).
 /// See [this issue comment](https://github.com/taiki-e/portable-atomic/issues/143#issuecomment-1866488569)
-/// for the known workaround.
+/// for a possible workaround with Rust stable or beta.
```

Comment:
```suggestion
/// for a workaround that works with stable and beta Rust channels.
```

I think this rewording should read a little better.